### ISSUE TITLE
Beware: Don't accidentally have multiple debugger windows open at once.

### ISSUE THREAD

**Issue by aleclarson** [facebook/react-native#3992]
If you find your log has multiple messages like this:

```
Running application "MyApp" with appParams: {"rootTag":1,"initialProps":{}}. __DEV__ === true, development-level warning are ON, performance optimizations are OFF
```

Check to make sure you don't have multiple `localhost:8081/debugger-ui` tabs open at the same time.

Cheers :smiley:

**Comment by jaygarcia** [comment]
does this need to be an open issue? If it's not an problem that needs to be resolved by the community, please close this :)

**Comment by aleclarson** [comment]
Perhaps there's a way to fix this issue from ever arising?

**Comment by martinbigio** [comment]
@aleclarson do you have consistent repro steps to get into this? I'd be happy to fix the problem if you provide them :)

**Comment by aleclarson** [comment]
@martinbigio
1. Open two tabs in Chrome; both pointing to `http://localhost:8081/debugger-ui`.
2. Enable "Debug in Chrome" on either a device or simulator.
3. Notice the device/simulator connects to both `debugger-ui` tabs.

If you can't reproduce with these steps, it's possible this has been fixed. I'm using React Native `v0.13.0` at the moment.

**Comment by tomauty** [comment]
It throws a different error related to callback IDs in later versions, though that might be a redux-logger problem. Think it's generally just bad practice to have multiple debuggers open...

**Comment by aleclarson** [comment]
> Think it's generally just bad practice to have multiple debuggers open...

Agreed, but this should be fixed to avoid the strange problems that arise when a debugger is unknowingly left open.

**Comment by martinbigio** [comment]
If the debugger is left unknowingly opened, shouldn't the existing tab be used instead of the app opening a new one?

**Comment by aleclarson** [comment]
@martinbigio Yeah. But if you manually open a debugger without knowing another debugger is open, you will face this error. It's a rare occurrence, but it happens.

**Comment by aleclarson** [comment]
This was fixed with #5715 :smile:
